### DISCUSSION TITLE
Can not display all figures when use R

### DISCUSSION THREAD

**Discussion by ZhouNan2020** [posit-dev/positron#5851] [Feedback]
When I use R language to draw fig (using ggplot), if the code I run contains multiple figs, the fig display area can only display the last fig, and cannot switch to other figs.

**Comment by juliasilge** [comment]
Can you create a [reprex](https://reprex.tidyverse.org/) (a minimal reproducible example) for this? The goal of a reprex is to make it easier for us to recreate your problem so that we can understand it and/or fix it. Often sharing a screenshot of what Positron is doing that is bad/wrong can be helpful. 

If you've never heard of a reprex before, you may want to start with the [tidyverse.org help](https://www.tidyverse.org/help/) page. Thanks! 🙌

**Reply by philibe** [reply]
@ZhouNan2020 :

It is the same behavior in RStudio IDE.
